.travis.yml

```diff
@@ -90,6 +90,10 @@ jobs:
             - python-bzrlib
             - python-paramiko
       script:
+        # Build once locally to download all the go modeule dependencies
+        - go run build/ci.go install
```

Comment:
This should be `go mod download`

Reply:
I've just tried it, but it downloads a whole lot more (116MB for `go install ./...` vs 156MB for `go mod download`). I guess the tests also depend on a lot of junk. Launchpad will need only the stuff `go install` needs, so might as well leave it as is.

Reply:
Actually `go install -n ./...` does the downloads but omits the actual compilation, so should be the best of both worlds.

Reply:
Added it as a workaround. Not sure if it works, need to merge the pr for it.